new/cookbook-adoption.md

```diff
@@ -0,0 +1,37 @@
+---
+RFC: unassigned
+Author: Nell Shamrell-Harrington <nshamrell@chef.com>
+Status: Draft
+Type: Process
+---
+
+# Cookbook Adoption On Supermarket
+
+## Motivation
+
+```
+    As a supermarket user,
+    When I find a cookbook,
+    And the cookbook is up for adoption,
+    And I want to adopt the cookbook,
+    And I request to adopt the cookbook, but get no response from the cookbook owner,
+    I want the ability to have ownership transferred to me,
+    So that I can update the cookbook
+```
+
+## Specification
+
+When a cookbook owner puts the cookbook up for adoption, that means they no longer wish to be the owner and maintainer of the cookbook.  When the owner puts the cookbook up for adoption, anyone who visits the cookbook's page on Supermarket will see a blue button that says "Adopt me!"  When a user clicks this button, an email is sent to the cookbook's current owner.  Occasionally, however, the cookbook's current owner does not respond to the email requests for adoption.
+
+If a user has requested to adopt a cookbook and has not received a response from the cookbook's current owner within 2 weeks, they should reach out to the Supermarket team via email (currently nshamrell@chef.io and rkidd@chef.io).  The Supermarket team will then email the current owner directly and make all other reasonable attempts to contact the owner.  If the Supermarket team does not receive a response from the current owner within a month, the Supermarket team will again attempt to contact the owner.  If the Supermarket team does not receive a response after attempting to contact the owner six times over six months, the Supermarket team will transfer ownership to the user who wishes to adopt the cookbok.
+
```

Comment:
How/where will forced transfers be logged?  How/where will they be announced?

I suggest having an open RFC in the [chef-rfc repository](https://github.com/chef/chef-rfc) that has a log of cookbooks that have their ownership changed via this process.  That log would show the date the adoption request was made, the dates contact was attempted, and the date the cookbook was transferred.  It would also need to show the previous and new owners.

What happens if multiple people request the same cookbook?